scipy/interpolate/fitpack2.py

```diff
@@ -1098,25 +1114,39 @@ class SmoothBivariateSpline(BivariateSpline):
 
     Parameters
     ----------
-    x, y, z : array_like
-        1-D sequences of data points (order is not important).
-    w : array_like, optional
-        Positive 1-D sequence of weights, of same length as `x`, `y` and `z`.
+    x : (N,) array_like
+        1-D array of independent input data. Must be increasing;
+        must be strictly increasing if `s` is 0.
+    y : (N,) array_like
+        1-D array of independent input data. Must be increasing;
+        must be strictly increasing if `s` is 0.
+    z : (N,N, ) array_like
+        2-D array of dependent input data, with the shape of
+        ``(x.size, y.size)``.
+    w : (N, ) array_like, optional
+        Positive 1-D sequence of weights, of same length as `x`, `y`, and `z`.
+        If `w` is None, weights are all equal. Default is None.
     bbox : array_like, optional
-        Sequence of length 4 specifying the boundary of the rectangular
-        approximation domain.  By default,
-        ``bbox=[min(x), max(x), min(y), max(y)]``.
-    kx, ky : ints, optional
-        Degrees of the bivariate spline. Default is 3.
+        4-sequence specifying the boundary of the rectangular
+        approximation domain. If `bbox` is None,
+        ``bbox=[min(x), max(x), min(y), max(y)]``. Default is None.
+    kx : int, optional
+        Degrees of the bivariate spline for values `x`. Default is 3.
+    ky : int, optional
+        Degrees of the bivariate spline for values `y`. Default is 3.
     s : float, optional
-        Positive smoothing factor defined for estimation condition:
-        ``sum((w[i]*(z[i]-s(x[i], y[i])))**2, axis=0) <= s``
-        Default ``s=len(w)`` which should be a good value if ``1/w[i]`` is an
-        estimate of the standard deviation of ``z[i]``.
+        Positive smoothing factor used to choose the number of knots. Number
+        of knots will be increased until the smoothing condition is satisfied::
+
+        sum((w[i]*(z[i]-spl(x[i], y[i])))**2, axis=0) <= s
```

Comment:
I am not sure this would render properly, or might be missing indentation.

Reply:
You are right!